django_extensions/management/commands/drop_test_database.py

```diff
@@ -136,8 +132,9 @@ def handle(self, *args, **options):
 
             try:
                 cursor.execute(drop_query)
-            except Database.ProgrammingError as e:
+            except (Database.ProgrammingError, Exception) as e:
```

Comment:
What's the reason for catching the very wide `Exception` ?
This makes `Database.ProgrammingError` not necessary anymore I think...
I'd rather not catch `Exception` if it's not absolutely necessary...

Reply:
ups, I put that general exception only for test purpose and I forget to remove it.

Reply:
Please commit and push and I will merge 👍